src/ctia/stores/es/crud.clj

```diff
@@ -282,25 +282,27 @@
         :or {all-of {} one-of {}}} :- FilterSchema
        ident
        params]
-      (let [filter-val (conj (q/prepare-terms all-of)
-                             (find-restriction-query-part ident))
+      (let [filter-val (cond-> (q/prepare-terms all-of)
+                         (restricted-read? ident)
+                         (conj (find-restriction-query-part ident)))
```

Comment:
same thing for find, could you add some tests related to `:authorized-anonymous` field in the identity?